### PR TITLE
BOM-2706: Django 3.2 Support

### PR DESCRIPTION
**Pull request by mzulqarnain1**
## Description

- Added Django 3.2 Support
- Applied Django Code Mods & Pyupgrade 
- Ran Make Upgrade

## Ticket Link

Link to the associated ticket
https://openedx.atlassian.net/browse/BOM-2706

### CODE REVIEW

File: openassessment/assessment/models/peer.py [hunk 1]
```diff
@@ -275,6 +275,9 @@ def get_waiting_step_details(
             )
         ).filter(
             graded_by_count__lt=must_be_graded_by
+        ).order_by(
```
Comment (mzulqarnain1) [inline openassessment/assessment/models/peer.py:278]:
```
openassessment/assessment/test/test_peer.py::TestPeerApi::test_get_waiting_step_details
openassessment/assessment/test/test_peer.py::TestPeerApi::test_get_waiting_step_details__peer_item_created_not_assessed
openassessment/xblock/test/test_staff_area.py::TestCourseStaff::test_waiting_step_details_api
  /home/travis/build/edx/edx-ora2/.tox/django30/lib/python3.8/site-packages/django/db/models/query.py:1261: RemovedInDjango31Warning: PeerWorkflow QuerySet won't use Meta.ordering in Django 3.1. Add .order_by('created_at', 'id') to retain the current query.```

Comment (mzulqarnain1) [reply openassessment/assessment/models/peer.py:278]: See https://app.travis-ci.com/github/edx/edx-ora2/jobs/529373300#L429 for logs.

**Comment by awais786** [comment]
@mzulqarnain1 plz make sandbox and do some testing.

**Comment by awais786** [comment]
@sarina Please review this PR.

**Review by UsamaSadiq — APPROVED** [review]

**Comment by mzulqarnain1** [comment]
> @mzulqarnain1 plz make sandbox and do some testing.

@awais786 Tested on the sandbox.
Screen Shot 2021-08-09 at 9 07 38 PM

**Comment by mzulqarnain1** [comment]
@edx/masters-devs-gta Please review this PR.

**Comment by natabene** [comment]
@mattcarter Could you please review this as soon as you can? This is in support of the Django upgrade project @jmbowman and I are doing. If you have concerns, please let me know.

**Comment by natabene** [comment]
@mzulqarnain1 Owning team is currently reviewing this and will provide feedback. CC @mattcarter

**Review by nsprenkle — APPROVED** [review]
LGTM

**Comment by natabene** [comment]
This will be merged on Thursday by the owning team unless they hear otherwise here.

**Pull request merged**